FWCore/ServiceRegistry/test/serviceregistry_t.cppunit.cpp

```diff
@@ -73,75 +73,64 @@ namespace {
 void testServiceRegistry::externalServiceTest() {
   edm::AssertHandler ah;
 
-  {
-    auto dummyPtr = std::make_unique<DummyService>();
-    dummyPtr->value_ = 2;
-    edm::ServiceToken token(edm::ServiceRegistry::createContaining(std::move(dummyPtr)));
-    {
-      edm::ServiceRegistry::Operate operate(token);
-      edm::Service<DummyService> dummy;
-      CPPUNIT_ASSERT(dummy);
-      CPPUNIT_ASSERT(dummy.isAvailable());
-      CPPUNIT_ASSERT(dummy->value_ == 2);
-    }
-    {
-      std::vector<edm::ParameterSet> pss;
-
-      edm::ParameterSet ps;
-      std::string typeName("DummyService");
-      ps.addParameter("@service_type", typeName);
-      int value = 2;
-      ps.addParameter("value", value);
-      pss.push_back(ps);
-
-      edm::ServiceToken token(edm::ServiceRegistry::createSet(pss));
-      edm::ServiceToken token2(
-          edm::ServiceRegistry::createContaining(std::move(dummyPtr), token, edm::serviceregistry::kOverlapIsError));
-
-      edm::ServiceRegistry::Operate operate(token2);
-      edm::Service<testserviceregistry::DummyService> dummy;
-      CPPUNIT_ASSERT(dummy);
-      CPPUNIT_ASSERT(dummy.isAvailable());
-      CPPUNIT_ASSERT(dummy->value() == 2);
-    }
-  }
+  {{auto dummyPtr = std::make_unique<DummyService>();
+  dummyPtr->value_ = 2;
+  edm::ServiceToken token(edm::ServiceRegistry::createContaining(std::move(dummyPtr)));
```

Comment:
This change is incorrect. the `token` is meant to have a lifetime that overlaps `token2` which is in the next scope.

Reply:
In the old version, "token" on line 79 is shadowing the "token" on
line 97.  I can change it to keep the token on line 79 alive if that
serves some test purpose internally. It wasn't obvious to me that
it was.

Reply:
Done. Fixed this so the token is still alive. Note this also had the side effect of fixing the weird formatting discussed above...